scene/gui/grid_container.cpp

```diff
@@ -259,6 +259,14 @@ int GridContainer::get_columns() const {
 	return columns;
 }
 
+int GridContainer::get_h_separation() const {
+	return theme_cache.h_separation;
+}
+
+int GridContainer::get_v_separation() const {
```

Comment:
I know it's for consistency, but the method is not used anywhere. It's internal only, so it doesn't need to exist for now.